playbooks/roles/jenkins/defaults/main.yml

```diff
@@ -21,11 +21,11 @@ jobs_disabled: false
 dist_tools:
   - acl
   - bzip2
-  - chromium-chromedriver
   - curl
   - fonts-liberation
   - git
   - gnupg2
+  - google-chrome-stable=92.0.4515.131-1
```

Comment:
I don't like that this is introducing a new version requirement, but the version of Chrome has to match the version of Chromedriver and since they're installed from two different places I couldn't think of a better way.